src/import/sst_service.rs

```diff
@@ -180,10 +183,25 @@ impl RequestCollector {
         self.accept(cf, m);
     }
 
+    /// check whether the unpacked size would exceed the max_raft_req_size after
+    /// accepting the modify.
+    fn should_send_batch_before_adding(&self, m: &Modify) -> bool {
+        let message_size = m.size() + WIRE_EXTRA_BYTES;
```

Comment:
maybe add some comments about the exact size between `Modify` and `Protobuf`

Reply:
See the comment in the `WIRE_EXTRA_BYTES`.